docs/using_lagoon/lagoon_yml.md

```diff
@@ -41,6 +41,11 @@ tasks:
 routes:
   insecure: Redirect
 
+production-backup-retention:
+  monthly: 1
+  weekly: 4
+  daily: 7
+
```

Comment:
the longer I look at this I would suggest to change it to this:
```
backup-retention:
  production:
    monthly: 1
    weekly: 4
    daily: 7
```
feels much better if in the future we also want to allow to change the backup-retention for `development` projects? (we don't have to define this right now)

Reply:
Updated the .lagoon.yml file definition and builddeploy code as requested.